test/NuGet.Server.V2.Tests/NuGetODataControllerTests.cs

```diff
@@ -238,73 +238,6 @@ public void FindPackagesByIdDoesNotHitBackendWhenIdIsEmpty()
             }
         }
 
-        public class SearchMethod
```

Comment:
Can we try to keep these in? Not really as unit tests but more to validate our OData interface returns correct info.